i18n/es/developer_guide.md

```diff
@@ -180,105 +180,105 @@ This Specification is distributed in the hope that it will be useful, but WITHOU
+
+## Metas
+
+C4 está hecho para ser un modelo óptimo de colaboración para proyectos de software de código abierto. Y se especifican los siguientes objetivos:
+
+- Maximizar la escala y la diversidad de la comunidad en torno a un proyecto, reduciendo la fricción para los nuevos colaboradores y creando un modelo de participación escalado con fuertes retroalimentaciones positivas;
+- Para aliviar las dependencias de individuos clave, separando diferentes conjuntos de habilidades para que haya un mayor grupo de competencia en cualquier dominio requerido;
+- Permitir que el proyecto se desarrolle de forma más rápida y precisa, aumentando la diversidad del proceso de toma de decisiones;
+- Apoyar el ciclo de vida natural de las versiones del proyecto, desde el experimental hasta el estable, permitiendo la experimentación segura, el fracaso rápido y el aislamiento del código estable;
+- Reducir la complejidad interna de los repositorios de proyectos, facilitando así la participación de los contribuyentes y reduciendo el margen de error;
+- Reforzar la propiedad colectiva del proyecto, lo que aumenta el incentivo económico a los contribuyentes y reduce el riesgo de secuestro por entidades hostiles.
+
+## Diseño
+
+### Preliminares
+
+- El proyecto USARÁ el sistema de control de revisión distribuido git.
+- El proyecto ESTARÁ alojado en github.com o equivalente, llamado "Plataforma".
+- El proyecto USARÁ utilizar la plataforma de Tracker de issues.
+- El proyecto DEBERÁ tener directrices claramente documentadas para el estilo de código.
+- Un "Colaborador" es una persona que desea proporcionar un parche, siendo un conjunto de compromisos que resuelven algún problema claramente identificado.
+- Un "Mantenedor" es una persona que fusiona parches al proyecto. Los mantenedores no son desarrolladores; Su trabajo es hacer cumplir el proceso.
+- Los colaboradores NO TENDRÁN acceso de commit al repositorio a menos que sean también Mantenedores.
+- Los mantenedores TENDRÁN tener acceso de commit al repositorio.
+- Todos, sin distinción ni discriminación, TENDRÁN el mismo derecho de convertirse en un Colaborador en los términos de este contrato.
+
+### Licencia y propiedad
+
+- El proyecto DEBE utilizar una licencia compartida, como la GPLv3 o una variante de la misma (LGPL, AGPL), o la MPLv2.
+- Todas las contribuciones al código fuente del proyecto ("parches") DEBERÁ utilizar la misma licencia que el proyecto.
+- Todos los parches son propiedad de sus autores. No habrá ningún proceso de asignación de derechos de autor.
+- Los derechos de autor del proyecto serán de propiedad colectiva de todos sus colaboradores.
+- Cada Contribuyente DEBERÁ ser responsable de identificarse en la lista de Colaboradores del proyecto.
+
+### Requisitos del parche
+
+- Mantenedores y contribuyentes DEBEN tener una cuenta de la plataforma y DEBERÍAN utilizar sus nombres verdaderos o un alias conocido.

```

Comment:
Suggestion:
"Los Mantenedores y los Contribuyentes DEBEN tener una cuenta **en** la plataforma y **DEBERÁN** usar sus nombres verdaderos o un alias conocido."

 * I noticed in this line that the word "contributor" has the translated word "contribuyente" (I have personally supported this translated word), and in the majority of the translated text, the word "colaborador = colaborator" is being used.